src/net/fail.rs

```diff
@@ -5,7 +5,6 @@ use warp::http::StatusCode;
 #[derive(Serialize)]
 struct Message {
 	code: u16,
-	#[serde(skip_serializing_if = "Option::is_none")]
```

Comment:
@rushmorem just wondering why you removed this line, but not line 9?

Reply:
Our HTTP engine only uses the `code`, `details` and `token` fields currently. I didn't touch the other fields because I didn't want to change the current behaviour on those fields unnecessarily. Would you like me to remove the line from the other fields too @tobiemh?

Removing those lines will potentially save someone some debugging time in future if we ever decide to use those fields too. `bincode` doesn't support deserialising from fields that don't exist. On the other hand, once we remove those lines, all errors will include those fields in JSON results even when they are just `null`.

Reply:
Ahh ok, let's leave it for the moment. Can always do it in a separate PR!